### PR TITLE
Add DoS auth models & initial auth app setup

### PR DESCRIPTION
**Pull request by mabe13**
This branch adds models from DoS to enable authorising that an api requestor can view and edit a service's capacity. 

In addition to this it also adds structure for a capacity auth django app and splits out the dos models into their own app for common use between the **capacity service** and **capacity auth apps**.

### CODE REVIEW

File: application/api/dos/models.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
+from django.db import models
+
+
+class Services(models.Model):
+
+    uid = models.CharField(max_length=255)
+    name = models.CharField(max_length=255)
+    # TODO Not used attributes of the services table, will be removed once model stability is confirmed
```
Comment (jope1) [inline application/api/dos/models.py]: I think we are stable enough now, lets remove these commented lines from the code.

File: application/api/dos/models.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+class Capacitystatuses(models.Model):
+    capacitystatusid = models.AutoField(primary_key=True)
+    color = models.CharField(max_length=255)
+
+    class Meta:
+        managed = False
+        db_table = "capacitystatuses"
+
+
+class Users(models.Model):
```
Comment (jope1) [inline application/api/dos/models.py]: Not sure we need this model.  Depends on how we are going to implement the authorisation though. Let's talk.

File: application/api/dos/models.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    approvedby = models.CharField(max_length=255, blank=True, null=True)
+    approveddate = models.DateTimeField(blank=True, null=True)
+    validationtoken = models.CharField(max_length=255, blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = "users"
+
+
+class Userservices(models.Model):
```
Comment (jope1) [inline application/api/dos/models.py]: Worth discussing, but I don't think we'll need this model (or the next 3).

File: application/api/dos/models.py (hunk 1 above)
Comment (mabe13) [reply application/api/dos/models.py]: Agree, will remove commented out lines

**Pull request merged**